modmesh/app/euler1d.py

```diff
@@ -545,14 +589,11 @@ def march_alpha2(self, steps=1):
             self.log(mm.time_registry.report())
         self.plt_mgr.update_lines()
 
-    def setup_solver(self, interval):
+    def setup_timer(self, interval):
         """
         :param interval: milliseconds
         :return: nothing
         """
-        self.shocktube.build_field(t=0)
```

Comment:
These solver initial flow is redundant, they are initialized in [line 624](https://github.com/solvcon/modmesh/pull/250/files#diff-4ebeb3148c683103bf030637fdf520154a4b12d9b660a82610861aa5250a0be1R624)